core/arch/arm/kernel/boot.c

```diff
@@ -1423,6 +1423,33 @@ static void init_primary(unsigned long pageable_part, unsigned long nsec_entry)
 	init_sec_mon(nsec_entry);
 }
 
+#ifdef CFG_SECURE_PARTITION
+static void check_crc32_feature(void)
+{
+	static bool checked;
+	static bool available;
+
+	/* Check once (i.e. on the boot core) and store the result */
+	if (!checked) {
+		available = feat_crc32_implemented();
+		checked = true;
+		return;
+	}
+
+	/* If available on the boot core, make sure other cores match */
+	if (available) {
```

Comment:
`if (available != feat_crc32_implemented())`

(assuming Jens below comment is addressed in some way)

Reply:
As discussed above, I removed this commit from the PR.